lightning/src/ln/peer_handler.rs

```diff
@@ -868,9 +916,7 @@ impl<Descriptor: SocketDescriptor, CM: Deref, RM: Deref, L: Deref> PeerManager<D
 
 	/// Process an incoming message and return a decision (ok, lightning error, peer handling error) regarding the next action with the peer
 	/// Returns the message back if it needs to be broadcasted to all other peers.
-	fn handle_message(&self, peer: &mut Peer, message: wire::Message) -> Result<Option<wire::Message>, MessageHandlingError> {
-		log_trace!(self.logger, "Received message {:?} from {}", message, log_pubkey!(peer.their_node_id.unwrap()));
```

Comment:
I don't know if this log message is important or not (looks useful but not critical to me). I couldn't find a nice way to keep it without putting a `Debug` constraint on the `T` in `Message` (which is maybe also fine).

Reply:
Ah, I didn't realize that is why it was needed. I think we want the constraint on `T` then.

Reply:
Added and restored the log message